build/esm.js

```diff
@@ -16,7 +16,7 @@ const localeTypePath = path.join(process.env.PWD, 'esm/locale', `index${typeFile
     readLocaleDir.forEach(async (l) => {
       const filePath = path.join(localeDir, l)
       const readFile = await promisify(fs.readFile)(filePath, 'utf8')
-      const result = readFile.replace("'dayjs'", "'../index'")
+      const result = readFile.replace(/'dayjs'/g, "'dayjs/esm'")
```

Comment:
seems `../index` should be the same file as `dayjs/esm`,  don't it?

Reply:
That’s for locale.
Plugins generated structure is different due to typings.
`translation/en.js` vs `plugins/duration/index.js`

Reply:
Feel free to commit this change. But it’s not broken.

Reply:
You can check the final file here https://unpkg.com/browse/dayjs@1.10.6/esm/locale/af.js

![image](https://user-images.githubusercontent.com/17680888/128672262-42b19d70-019e-4317-8df3-4ad734539a3b.png)


`../index` refers to `esm/index.js`, which I think it's correct?